diff -Naur --exclude .git --exclude config.h st.orig/config.def.h st/config.def.h
--- st.orig/config.def.h	2019-10-09 23:19:06.068637595 +0200
+++ st/config.def.h	2019-10-09 23:23:07.296183085 +0200
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "DejaVu Sans Mono:size=10";
 static int borderpx = 2;
 
 /*
@@ -84,31 +84,34 @@
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	"#181818",  // 00 ~ base00 ~ black
+	"#ab4642",  // 01 ~ base08 ~ red
+	"#a1b56c",  // 02 ~ base0B ~ green
+	"#f7ca88",  // 03 ~ base0A ~ yellow
+	"#7cafc2",  // 04 ~ base0D ~ blue
+	"#ba8baf",  // 05 ~ base0E ~ magenta
+	"#86c1b9",  // 06 ~ base0C ~ cyan
+	"#d8d8d8",  // 07 ~ base05 ~ white
+
+	"#585858",  // 08 ~ base03 ~ black
+	"#ab4642",  // 09 ~ base08 ~ red
+	"#a1b56c",  // 10 ~ base0B ~ green
+	"#f7ca88",  // 11 ~ base0A ~ yellow
+	"#7cafc2",  // 12 ~ base0D ~ blue
+	"#ba8baf",  // 13 ~ base0E ~ magenta
+	"#86c1b9",  // 14 ~ base0C ~ cyan
+	"#f8f8f8",  // 15 ~ base07 ~ white
+
+	"#dc9656",  // 16 ~ base09
+	"#a16946",  // 17 ~ base0F
+	"#282828",  // 18 ~ base01
+	"#383838",  // 19 ~ base02
+	"#b8b8b8",  // 20 ~ base04
+	"#e8e8e8",  // 21 ~ base06
 
 	[255] = 0,
 
 	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
 };
 
 
@@ -118,8 +121,8 @@
  */
 unsigned int defaultfg = 7;
 unsigned int defaultbg = 0;
-static unsigned int defaultcs = 256;
-static unsigned int defaultrcs = 257;
+static unsigned int defaultcs = 15;
+static unsigned int defaultrcs = 8;
 
 /*
  * Default shape of cursor
diff -Naur --exclude .git --exclude config.h st.orig/x.c st/x.c
--- st.orig/x.c	2019-10-09 23:19:06.068637595 +0200
+++ st/x.c	2019-10-05 22:56:53.507162014 +0200
@@ -712,7 +712,7 @@
 	XRenderColor color = { .alpha = 0xffff };
 
 	if (!name) {
-		if (BETWEEN(i, 16, 255)) { /* 256 color */
+		if (BETWEEN(i, 22, 255)) { /* 256 color */
 			if (i < 6*6*6+16) { /* same colors as xterm */
 				color.red   = sixd_to_16bit( ((i-16)/36)%6 );
 				color.green = sixd_to_16bit( ((i-16)/6) %6 );
